generic/tests/trans_hugepage.py

```diff
@@ -64,33 +64,33 @@ def get_mem_status(params, role):
     logging.info("Smoke test start")
```

Comment:
BTW `logging.info` needs to be replaced with `test.log.info`

Reply:
Thanks for raising this issue. As  @luckyh  have update all of this kind of issues and patches under review,  let's leave this issue with this patch.